node-data/src/message.rs

```diff
@@ -561,55 +554,61 @@ pub mod payload {
             })
         }
     }
-    #[derive(Clone, Copy, Default)]
+    #[derive(Clone, Copy, Debug, Default)]
     #[cfg_attr(
         any(feature = "faker", test),
         derive(fake::Dummy, Eq, PartialEq)
     )]
     pub enum QuorumType {
-        /// Quorum on Valid Candidate
-        ValidQuorum = 0,
-        // Quorum on Invalid Candidate
-        InvalidQuorum = 1,
-        //Quorum on Timeout (NilQuorum)
-        NilQuorum = 2,
-        // NoQuorum
+        /// Supermajority of Valid votes
+        Valid = 0,
+        /// Majority of Invalid votes
+        Invalid = 1,
+        /// Majority of NoCandidate votes
+        NoCandidate = 2,
+        /// No quorum reached (timeout expired)
         #[default]
         NoQuorum = 255,
     }
 
     impl From<u8> for QuorumType {
         fn from(v: u8) -> QuorumType {
             match v {
-                0 => QuorumType::ValidQuorum,
-                1 => QuorumType::InvalidQuorum,
-                2 => QuorumType::NilQuorum,
+                0 => QuorumType::Valid,
+                1 => QuorumType::Invalid,
+                2 => QuorumType::NoCandidate,
                 _ => QuorumType::NoQuorum,
             }
         }
     }
 
-    impl fmt::Debug for QuorumType {
-        fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-            let label = match self {
-                QuorumType::ValidQuorum => "valid_quorum",
-                QuorumType::InvalidQuorum => "invalid_quorum",
-                QuorumType::NilQuorum => "nil_quorum",
-                QuorumType::NoQuorum => "no_quorum",
-            };
-            f.write_str(label)
-        }
-    }
-
     #[derive(Debug, Clone, Default)]
     #[cfg_attr(
         any(feature = "faker", test),
         derive(fake::Dummy, Eq, PartialEq)
     )]
     pub struct ValidationResult {
-        pub quorum: QuorumType,
-        pub vote: Vote,
-        pub sv: StepVotes,
+        pub(crate) quorum: QuorumType,
+        pub(crate) vote: Vote,
```

Comment:
Not part of this PR, but I having both `quorum` and `vote` is redundant. Conceptually we only need `quorum`, but I would personally add `NoQuorum` to Vote (indeed, this is a valid vote for Ratification) and keep only `vote` here.
Just commenting to keep it in mind

Reply:
That will be tackled while implementing #1268 

ps: Vote::NoQuorum has already been added, with a check ensuring that vote cannot be collected during Validation